src/main/views/MattermostBrowserView.ts

```diff
@@ -463,26 +490,6 @@ export class MattermostBrowserView extends EventEmitter {
      * WebContents event handlers
      */
 
-    private handleDidFinishLoad = () => {
```

Comment:
Nope, this was the weird code I had to do before to make sure the renderer process had the viewId to send up. Since we just work with the `webContentsId` now, this is unnecessary :)